README.md

```diff
@@ -5,13 +5,23 @@
 
 Scripts used to create a system image for SR brain board SD cards.
 
+**Supported Platforms:**
+
+| Platform           | Status    |
+|--------------------|-----------|
+| ODroid U3          | Supported |
+| Raspberry Pi 3     | Supported |
+| Raspberry Pi 4/400 | Supported |
```

Comment:
```suggestion
| Raspberry Pi 4B/400 | Supported |
```

Reply:
Interestingly there's no 4A

Reply:
Yes, but we can't guarantee that there won't be in the future.

Reply:
I've never even heard of it being referred to as the 4B, although the board does say `Raspberry Pi 4 Model B`